kratos/python_scripts/point_output_process.py

```diff
@@ -174,7 +172,7 @@ def ExecuteFinalize(self):
 
     def __CheckVariableIsSolutionStepVariable(self, var):
         # if the requested Variable is a component we check the source Variable
-        if type(var) == KratosMultiphysics.Array1DComponentVariable:
+        if type(var) == KratosMultiphysics.DoubleVariable:
```

Comment:
hm note to myself, check this bcs I think as is it is broken
```suggestion
        if type(var) == KratosMultiphysics.DoubleVariable: # TODO check this, might no longer work!
```